### ISSUE TITLE
Disallow comments in the import section

### ISSUE THREAD

**Issue by robertwhitebit** [sevntu-checkstyle/sevntu.checkstyle#373]
A reason to use comments in the (static)import section, is, to comment out imports. However, imports are managed automatically by the IDE and therefore will be inserted/remove automatically at the correct position. So there is no need to comment them out.

Moreover, the Android Studio formatter handles import comments in an interesting way.

- Commented out imports will be removed automatically (if they aren't before or after the import section).

Before running the formatter:
```java
import com.puppycrawl.tools.checkstyle.api.Check;
// import com.puppycrawl.tools.checkstyle.api.DetailAST;
import com.puppycrawl.tools.checkstyle.api.TokenTypes;
```
After running the formatter:
```java
import com.puppycrawl.tools.checkstyle.api.Check;
import com.puppycrawl.tools.checkstyle.api.TokenTypes;
```

Before:
```java
package abc;

// import com.puppycrawl.tools.checkstyle.api.Check;
import com.puppycrawl.tools.checkstyle.api.DetailAST;
import com.puppycrawl.tools.checkstyle.api.TokenTypes;
```

After:
```java
package abc;

// import com.puppycrawl.tools.checkstyle.api.Check;

import com.puppycrawl.tools.checkstyle.api.DetailAST;
import com.puppycrawl.tools.checkstyle.api.TokenTypes;
```

- "Normal" comments will be moved to the bottom if they are inside the import block:

Before:
```java
import com.puppycrawl.tools.checkstyle.api.Check;
import com.puppycrawl.tools.checkstyle.api.DetailAST; // Needed?
import com.puppycrawl.tools.checkstyle.api.TokenTypes;

class Abc...
```

After:
```java
import com.puppycrawl.tools.checkstyle.api.Check;
import com.puppycrawl.tools.checkstyle.api.DetailAST;
import com.puppycrawl.tools.checkstyle.api.TokenTypes;

// Needed?

class Abc...
```

**Comment by robertwhitebit** [comment]
The eclipse optimize import functionality does *not* remove import comments, but does also move comments to the bottom as described above.

Before:
```java
import com.oracle.net.Sdp;
//import com.oracle.nio.BufferSecretsPermission;
import com.oracle.util.Checksums;

public final class Utils {
```
After:
```java
import com.oracle.net.Sdp;
import com.oracle.util.Checksums;
//import com.oracle.nio.BufferSecretsPermission;

public final class Utils {
```

**Comment by mkordas** [comment]
@robertwhitebit - it's great idea for a check. 

Generally detecting commented out code is hard task and may generate some false-positives if e.g. comment contains some instructions how to use the code.

Here I think the check would not be controversial at all. If comment begins with 'import' and is somewhere above first class in file, then we can raise the violation.

@romani, do you agree with me?

**Comment by romani** [comment]
I would rather postpone commented code detection for future, it is not that simple, sonar have rule for that, but even them have false positives

**Comment by mkordas** [comment]
So that's what I mean - instead of creating controversial and hard to implement rule for commended out code we can add very simple rule to disallow commended out imports to start with.
